src/spec/tests/core/closecountrydropdownEvent.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+describe("close:countrydropdown event:", function() {
+
+  var spy;
+
+  beforeEach(function() {
+    intlSetup();
+    input = $("<input>");
+    spy = spyOnEvent(input, 'close:countrydropdown');
+  });
+
+  afterEach(function() {
+    input.intlTelInput("destroy");
+    input = null;
+  });
+
+  describe("init plugin", function() {
+
+    beforeEach(function() {
+      input.intlTelInput();
+    });
+
+    it("does not trigger the event", function() {
+      expect(spy).not.toHaveBeenTriggered();
+    });
+
+    it("clicking outside the dropdown to close it triggers the event", function() {
+      openCountryDropDown();
+      $("<h1>").click();
```

Comment:
I'm not sure if this will work, if the element is not in the DOM? Instead, let's inject the input into the DOM in the beforeEach (see example [here](https://github.com/jackocnr/intl-tel-input/blob/master/src/spec/tests/core/dropdownShortcuts.js#L7), and remove it in the afterEach) and then use `input.click()` - does that work?